renovate.json

```
@@ -0,0 +1,62 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    "config:base",
+    ":gitSignOff"
+  ],
+  "packageRules": [
```

I'd say start small and only enable the managers that we've tested, and only update selected dependencies. That gives us a way to migrate stuff from dependabot to Renovate if we want to, without having both of them doing the same stuff:

```suggestion
  "enabledManagers": [
    "gomod",
    "custom.regex"
  ],
  "packageRules": [
    {
      "description": "Ignore all Go dependencies by default",
      "enabled": false,
      "matchManagers": [
        "gomod"
      ]
    },
```